Components/typography.css

```diff
@@ -0,0 +1,113 @@
+.banner1,
```

Comment:
instead of writing banner1, banner2, you can add another common class to all the components where you want these styles. Will look cleaner.

Reply:
I can do that but then these classes wont be stand alone i.e. every time user must give two classes for it. Am i missing something? Can you give me an example.

Reply:
Yeah, so you can do this :
<div class=" banner banner1">b1.Banner</div>
<div class=" banner banner2">b2.Banner</div>

And then add these styles for banner instead of doing it for banner1,banner2.
And in my opinion, it is okay to add two classes, one generic and one component-specific, if they have some common styling.

Reply:
Sure. Thanks for the inputs